app/views/layouts/admin.html.erb

```diff
@@ -44,13 +44,16 @@
     <div class="admin-footer">
       <div class="container">
         <div class="row">
-          <span class="span4 offset8">
+          <span class="span6 offset6">
             <ul class="nav nav-pills">
               <li>
                 <a href="http://alaveteli.org/docs/running/admin_manual/">
                   Admin Manual
                 </a>
               </li>
+              <li>
+                <%= link_to 'Alaveteli Changelog', admin_changelog_index_path %>
```

Comment:
I wonder about making this less "technical" sounding?

Could use "New Features", though that kind of only makes sense for the ~week or so after upgrade.

Reply:
Are there always new features, or will it sometimes be bug fixes? How about "Latest changes" or "Recent changes"?

Reply:
Yeah it varies. Sometimes a release only includes translation updates. I quite like "Latest changes" or "Recent changes" as more human friendly, but I wonder if that's too general and could mean e.g. on-site content changes like "new requests" or whatever. Could call it "Software Updates" to make it align more with operating systems, which is quite universally understood?

Reply:
Went with "Changelog" after some discussion about what we do on other mySociety services. Also added some intro description when you click through to explain what's been viewed.